### PR TITLE
Getting item list from the game itself

### PR DESCRIPTION
**Pull request by DaddyWesker**
So now we'll be able to get item list from the game itself instead of getting them from minecraft_data. 

I'll also create PR to minecraft_demo to test these changes.

### CODE REVIEW

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromItemsImplementation.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                !comm[1].equalsIgnoreCase("off")) {
+            this.sendRec = true;
+            return true;
+        }
+        if (comm.length == 2 && comm[0].equalsIgnoreCase(ObservationFromItem.ITEM_LIST.value()) &&
+                comm[1].equalsIgnoreCase("off")
+        ) {
+            this.sendRec = false;
+            return true;
+        }
```
Comment (noskill) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromItemsImplementation.java]:
```suggestion
        if (comm.length == 2 && comm[0].equalsIgnoreCase(ObservationFromItem.ITEM_LIST.value())){
          if (comm[1].equalsIgnoreCase("off")) {
               this.sendRec = false;
           } else {
               this.sendRec = true;
           }
          return true;
        }
```

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromItemsImplementation.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        Registry<Item> str_ent = MinecraftClient.getInstance().world.getRegistryManager().get(ITEM_KEY);
+        List<Item> list_ent = str_ent.stream().toList();
+        JsonArray items = new JsonArray();
+        for (Item ent: list_ent)
+        {
+            String item_name = ent.toString();
+            items.add(item_name);
+        }
+        json.add("item_list", items);
```
Comment (noskill) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromItemsImplementation.java:69]: Does it contain blocks or just items that might be crafted?

Comment (DaddyWesker) [reply src/main/java/io/singularitynet/MissionHandlers/ObservationFromItemsImplementation.java:69]: @noskill  actually, I'm not sure. List does contain some block-like things like flower_pot, air, oak_sign, but not [Acacia Wall Sign](https://minecraft.fandom.com/wiki/Acacia_Wall_Sign) for example. But it contains craftable items like pickaxes, iron_ingot.

**Review by Necr0x0Der — APPROVED** [review]

**Comment by Necr0x0Der** [comment]
Seems to work for me.

**Pull request merged**